src/ActionSheetProvider.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import ActionSheet from './ActionSheet';
+import { ActionSheetOptions } from './types';
+
+export type Context = {
+  showActionSheetWithOptions:
+    | ((options: ActionSheetOptions, callback: (i: number) => void) => void)
+    | null;
+};
+
+const { Provider, Consumer } = React.createContext<Context>({
```

Comment:
@brentvatne is it still valuable to use [create-react-context](https://github.com/jamiebuilds/create-react-context) for react native apps?

Reply:
not necessary these days i think, people should mostly be on a version of react-native with React.createContext. we just moved react-navigation away from it